src/screens/WaitTimeScreen.tsx

```diff
@@ -1,42 +1,449 @@
+    setRefreshing(true);
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 1000);
+  }, []);
+
+  const modalComponent = (
+    <Modal
+      transparent={false}
+      visible={isVisible}
+      animationType="slide-up"
+      popup
+      style={styles.modal}
+    >
+      <TouchableWithoutFeedback onPress={() => setIsVisible(false)}>
+        <View style={styles.modalBackground}>
+          <TouchableWithoutFeedback>
+            <View style={styles.modalHeader}>
+              <View style={styles.modalActionRow}>
+                <TouchableOpacity
+                  onPress={() => {
+                    setIsVisible(false);
+                  }}
+                >
+                  <StyledText style={styles.modalButtons}>Cancel</StyledText>
+                </TouchableOpacity>
+                <StyledText>Sort By</StyledText>
+                <TouchableOpacity
+                  onPress={() => {
+                    setIsVisible(false);
+                  }}
+                >
+                  <StyledText style={styles.modalButtons}>Done</StyledText>
+                </TouchableOpacity>
+              </View>
+              <View>
+                <Button
+                  style={styles.sortButtons}
+                  onPress={() => {
+                    setIsVisible(false);
+                    setSortBy(SortByEnum.TIME);
+                  }}
+                >
+                  <StyledText>Time</StyledText>
+                  <View style={styles.checkIconTime}>
+                    {sortBy === "Time" && (
+                      <Feather name="check" size={24} color="black" />
+                    )}
+                  </View>
+                </Button>
+                <Button
+                  style={styles.sortButtons}
+                  onPress={() => {
+                    setIsVisible(false);
+                    setSortBy(SortByEnum.DISTANCE);
+                  }}
+                >
+                  <StyledText>Distance</StyledText>
+                  <View style={styles.checkIconDistance}>
+                    {sortBy === "Distance" && (
+                      <Feather name="check" size={24} color="black" />
+                    )}
+                  </View>
+                </Button>
+              </View>
+            </View>
+          </TouchableWithoutFeedback>
+        </View>
+      </TouchableWithoutFeedback>
+    </Modal>
+  );
+
   return (
-    <View style={styles.container}>
-      <StyledText style={styles.huge}>Wait Times</StyledText>
-      <WhiteSpace />
-      <Button
-        type={"primary"}
-        onPress={() =>
-          navigation.navigate(LOCATION_DETAILS, { location: "Location X" })
-        }
-      >
-        <StyledText>Go to Location X</StyledText>
-      </Button>
-      <StatusBar style="auto" />
-    </View>
+    <TouchableWithoutFeedback
+      onPress={() => {
+        searchBar?.inputRef?.blur();
+      }}
+    >
+      <View style={theme.styles.screenContainer}>
```

Comment:
You can add the bottomPadding removal here.
```suggestion
      <View style={[theme.styles.screenContainer, {bottomPadding: 0}]}>
```